modules/integration_azure-functions/conf/01-function-errors.yaml

```diff
@@ -0,0 +1,28 @@
+
+module: Azure Functions
+name: "wrapper errors"
+id: errors
+signals:
+  errors:
+    metric: azure.function.errors
+    extrapolation: "last_value"
+    rollup: "average"
+  invocations:
+    metric: azure.function.invocations
+    extrapolation: "last_value"
+    rollup: "average"
```

Comment:
I would use `sum` in case of concurrent datapoints on same MTS

Reply:
fixed in last commit.